CardScan/Classes/ScanViewController.swift

```diff
@@ -31,6 +31,10 @@ import Vision
     @objc func skipButton() -> String
 }
 
+@objc public protocol TestingImageDataSource {
```

Comment:
Can you give me an example of how this protocol will be used? I read below that it can be used for custom testing but I am still unclear on how it it should be implemented.

Reply:
Yeah, checkout `Example/CardScan/ViewController.swift`, which is part of this PR. It shows how to implement this protocol and how to use it.